doc/api.md

```diff
@@ -475,6 +475,15 @@ The listener will receive an object with the following structure:
 
 * **suspendDetected** - event notifications about detecting suspend event in host computer.
 
+* **reportAProblemPressed** - event notifications fired when the user presses the `Report a problem` button on the feedback submission dialog.
+This must go in tandem with `SHOW_REPORT_A_PROBLEM` flag from `interfaceConfigOverwrite` being set to true, otherwise the `Report a problem` button would
+not being shown so the handler will never be called on the consumer side. The listener will receive an object with the following structure:
+```javascript
+{
+    roomUrl: string // the href of the conference
```

Comment:
I'm wondering why do we need this property? I think it is known by the consumer of the API anyway, isn't it?